actiontext/lib/action_text/content.rb

```diff
@@ -110,6 +110,9 @@ def render_attachment_galleries(&block)
     #
     #   content = ActionText::Content.new("<h1>Funny times!</h1>")
     #   content.to_plain_text # => "Funny times!"
+    #
+    #   content = ActionText::Content.new("<div onclick='action()'>safe<script>unsafe</script></div>")
+    #   content.to_plain_text # => "safeunsafe"
```

Comment:
While we're here, I'm a little uncomfortable with the docstring headline for this method's description, can we update it?

Currently it says

```
Returns the content as plain text with all HTML tags removed.
```

Which I don't think is the best way to phrase this, since

```ruby
ActionText::Content.new("&lt;script&gt;alert()&lt;/script&gt;").to_plain_text
# => "<script>alert()</script>"
```

Can I ask that we modify this description and the one in `actiontext/app/models/action_text/rich_text.rb` to something like:

```
Returns a plain-text version of the markup contained by the document,
with tags removed but HTML entities encoded.

NOTE that the returned string is not HTML safe and should not be
rendered in browsers.
```

Reply:
Good point! I wasn't aware of this.
I think it would help if we add the example to explain this.
Moving the `NOTE` to the bottom, followed by the example:
```ruby
#  NOTE that the returned string is not HTML safe and should not be rendered in browsers.
#    ActionText::Content.new("&lt;script&gt;alert()&lt;/script&gt;").to_plain_text
# => "<script>alert()</script>"
```

Reply:
Done!